mala/losses/malis_loss.py

```diff
@@ -17,10 +17,14 @@ def get_edge_weights(self, affs, gt_affs, gt_seg):
         weights_neg = self.malis_pass(affs, gt_affs, gt_seg, pos=0)
         weights_pos = self.malis_pass(affs, gt_affs, gt_seg, pos=1)
 
-        return weights_neg + weights_pos
+        return (weights_neg, weights_pos)
 
     def malis_pass(self, affs, gt_affs, gt_seg, pos):
 
+        # in the positive pass (pos==1), we set boundary edges (gt_aff==0) to their true values
+        # in the negative pass (pos==0), we set non-boundary edges (gt_aff==1) to their true values
+        affs[gt_affs == (1-pos)] = 1-pos
```

Comment:
This unintentionally changes the passed predictions (passed as reference, not copy).